### ISSUE TITLE
animation-bug: unwanted attributes with from/to

### ISSUE THREAD

**Issue by bigmistqke** [LXSMNSYC/solid-styled#15]
animations are currently not working with `css`, because unwanted attributes are being added to the stylesheet.

from the solidjs-template (npx degit solidjs/templates/js my-app), the animation of the logo gets compiled to:
```css
@keyframes c-Anonymous-c8d13f94-0-logo-spin{from[s\:c-Anonymous-c8d13f94-0]{transform:rotate(0deg)}to[s\:c-Anonymous-c8d13f94-0]{transform:rotate(360deg)}}
```
while it should be
```css
@keyframes c-Anonymous-c8d13f94-0-logo-spin{from{transform:rotate(0deg)}to{transform:rotate(360deg)}}
```

**Comment by lxsmnsyc** [comment]
Seems like the body of keyframes are being treated as style rules by the parser.

**Comment by lxsmnsyc** [comment]
Should be fixed in 0.7.5

**Comment by bigmistqke** [comment]
nice, works on my end too. 
thanks alexis 👍
